### DISCUSSION TITLE
Resizing?

### DISCUSSION THREAD

**Discussion by nanodeath** [TokisanGames/Terrain3D#293] [Questions] [answered]
I'm exporting a map with settings that looks like this in World Machine:

![image](https://github.com/TokisanGames/Terrain3D/assets/10625/277f6099-6973-4d30-8a60-608a62db6382)

Notably, it's 4096x4096, but also represents a 2km by 2km. When I import into Terrain3D, it creates a region that's 4km by 4km. If I try to scale down the Node3D, nothing happens.

I assume I need to tweak the shader? I can't really correct it on the World Machine side because it's impractical to resize everything every time I want to change the output resolution.

**Comment by nanodeath** [comment, marked as answer]
Actually, is this a dupe of https://github.com/TokisanGames/Terrain3D/issues/77? edit: going to close, looks very similar.

**Reply by TokisanGames** [reply]
It is a duplicate of #131.

**Reply by nanodeath** [reply]
Ah, you're right.
